src/spec-node/devContainersSpecCLI.ts

```diff
@@ -184,6 +184,8 @@ async function provision({
 		remoteEnv: keyValuesToRecord(addRemoteEnvs),
 		additionalCacheFroms: addCacheFroms,
 		useBuildKit: buildkit,
+		buildxPlatform: undefined,
```

Comment:
Could we add tests that exercise the `--platform` option?

Tests for `--push` might need a bit more thought as we need a registry to push to?

Reply:
I was looking into that, but couldn't find any utests that exercises the args (i.e `cache-from, buildkit`... ). Any pointers or suggestions are welcome.

Reply:
nvm, I found some at `cli.test.ts`

Reply:
Unresolving as I can't see any added tests 😄

Reply:
@stuartleeks , @chrmarti , latest [commit](https://github.com/devcontainers/cli/pull/24/commits/cc6b381972409434e2d9052fab4942a5c8cfaba2) has utest. But seems like buildkit is not set, so I was wondering if any of you can give me some hints here, since the test that needs to fail, it is not. Thanks for your help.